### PR TITLE
 New variables in the player script

### PR DESCRIPTION
**Pull request by tsMezotic**
added 2 more variables in the player script for faster editing, like walkspeed and jumpstrength/height
too lazy to do any more

### CODE REVIEW

**Review by YaManTarzan — COMMENTED** [review]
Good for now but velocity should problably be added to insread of set

**Review by YaManTarzan — APPROVED** [review]

File: scripts/player.gd [hunk 1]
```diff
@@ -3,22 +3,25 @@ var velocity : Vector2 = Vector2(0,0)
 var up : Vector2 = Vector2(0,-1)
 var health : float = 100.0
 
+var walkspeed = 150
```
Comment (keri4141) [inline scripts/player.gd]: camelCase for variable names please.

Rename to walkSpeed

Also might be better to export the variable

export var walkSpeed

That way u can just modify it on the Editor and not have to dig thru the code each time

File: scripts/player.gd [hunk 2]
```diff
@@ -3,22 +3,25 @@ var velocity : Vector2 = Vector2(0,0)
...
+var jumpstrength = 150
```
Comment (keri4141) [inline scripts/player.gd]: Same with this,

camelCase and export

**Review by keri4141 — CHANGES_REQUESTED** [review]
camelCase please and for this particular case can you export the variables

**Review by keri4141 — APPROVED** [review]

**Pull request merged**